ts/api/api.ts

```diff
@@ -123,6 +124,11 @@ export interface OpenYoloApi {
    * Cancels the last pending OpenYOLO request.
    */
   cancelLastOperation(): Promise<void>;
+
+  /**
+   * Sets the desired log level for the OpenYOLO library.
+   */
+  setLogLevel(level: LogLevel): Promise<void>;
```

Comment:
I wasn't envisioning the log level to be set in such way, and I thought we had agreed on a different solution :/

All the "configuration" methods of OpenYolo currently lives on the `OnDemandOpenYoloApi` interface (`setTimeouts`, `setRenderMode`). The `OpenYoloApi` interface is reserved for credentials operations. IMO, we should not pollute the OpenYoloApi with a configuration method.
I thought you would set it on the `OnDemandOpenYoloApi` interface, and send another query parameter to the provider iframe URL (alongside `client` and `renderMode`) so that the `ProviderFrame` would initialize with the specified log level (or the default one). See our provider implementation where we pass the different parameters. I thought that's what we had discussed yesterday... What benefit do you see in allowing changing the log level dynamically?

The query parameter solution would avoid the need for another "message" and reduce the code changes required. What do you think?